examples/angular-example/.vscode/launch.json

```
@@ -0,0 +1,20 @@
+{
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comments aren't allowed this way in JSON. Is it possible to use either one of these two solutions instead?
- [`JSON5`](https://json5.org/)
- using a hack to define this like this (does the functionality still work in VS Code?):
```suggestion
  "_for_more_information_visit": "https://go.microsoft.com/fwlink/?linkid=830387"
```